src/components/NovelList.tsx

```diff
@@ -37,6 +45,22 @@ const NovelList: React.FC<
     }
   }, [isListView, orientation, novelsPerRow]);
 
+  var extendedNovelList: Array<listDataItem> = props?.data;
+  if (props.data?.length && props.inPlugin) {
+    let remainder = numColumns - (props.data?.length % numColumns);
+    let extension: Array<listDataItem> = [];
+    if (remainder !== 0 && remainder !== numColumns) {
+      extension.push({
+        cover: '',
+        name: '',
+        path: 'loading-' + remainder,
+        completeRow: -remainder,
```

Comment:
why negative number?

Reply:
This is just to prevent key issues when the skeleton for the novel